src/integrations/admin/indexation-integration.php

```diff
@@ -162,6 +172,11 @@ public function register_hooks() {
 	 * @return void
 	 */
 	public function enqueue_scripts() {
+		// If we are in a non-production environment, don't index the indexables.
```

Comment:
I would remove this comment, the method name is descriptive enough and if `should_index_indexables` is changed because other decisions are made there, you have to update this comment as well.